engines/registry/app/models/registry/authenticated_client.rb

```diff
@@ -22,7 +22,6 @@ def authenticate_by_system_credentials(login, password)
       @account = login
       @auth_strategy = :system_credentials
     end
-    @auth_strategy
```

Comment:
Why use the class variable at all?

The condition in line 9-10 could be changed to 

```
if authenticate_by_system_credentials(login, password)
```

and then you don't need to instantiate and pass around the class variable.

Reply:
Resolved in 06cd9719